From cec37f79f3c0cf3ce324635f7aaf945dce725ef8 Mon Sep 17 00:00:00 2001
From: Matija Parausic <matija@Matijas-Mac-mini-2032.local>
Date: Thu, 12 Jun 2014 17:16:06 +0200
Subject: [PATCH] qqq

---
 qqqq | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 qqqq

diff --git a/qqqq b/qqqq
new file mode 100644
index 0000000..811feb5
--- /dev/null
+++ b/qqqq
@@ -0,0 +1 @@
+qqqq
-- 
1.7.12.4 (Apple Git-37)

